perfkitbenchmarker/linux_benchmarks/netperf_pps_benchmark.py

```diff
@@ -147,43 +147,38 @@ def RunNetperfAggregate(vm, server_ips):
   """
 
   # setup remote hosts file
-  vm.RemoteCommand("cd %s && rm remote_hosts"
-                   % (netperf.NETPERF_EXAMPLE_DIR))
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && rm remote_hosts")
   ip_num = 0
   for ip in server_ips:
-    vm.RemoteCommand("cd %s && echo 'REMOTE_HOSTS[%d]=%s' >> remote_hosts"
-                     % (netperf.NETPERF_EXAMPLE_DIR, ip_num, ip))
+    vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && "
+                     f"echo 'REMOTE_HOSTS[{ip_num}]={ip}' >> remote_hosts")
     ip_num += 1
 
-  vm.RemoteCommand("cd %s && echo 'NUM_REMOTE_HOSTS=%d' >> remote_hosts"
-                   % (netperf.NETPERF_EXAMPLE_DIR, len(server_ips)))
-  vm.RemoteCommand('cd %s && export PATH=$PATH:.'
-                   % (netperf.NETPERF_EXAMPLE_DIR))
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && "
+                   f"echo 'NUM_REMOTE_HOSTS={len(server_ips)}' >> remote_hosts")
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && export PATH=$PATH:.")
```

Comment:
Side comment: I have no idea what this line is trying to do.  SSH in to cd to a directory and then export a path?  That doesn't do anything useful.

Reply:
You're right. I'm not sure why that was still in there.